broker/utils/plugins/k8s.py

```diff
@@ -63,7 +63,13 @@ def create_job(app_id, cmd, img, init_size, env_vars,
         )
     )
 
+    resources = kube.client.V1ResourceRequirements(
+        limits=kwargs['limits'] if kwargs.get('limits') else None,
+        requests=kwargs['requests'] if kwargs.get('limits') else None
```

Comment:
Shouldn't these 2 be retrieved from different keys of `kwargs`? 
Right now they're both being retrieved from `limits` key.
Also, `kwargs.get(key)` already returns None for non-existing keys, so `if kwargs.get('limits') else None` is redundant.

Reply:
I think it could be even more transparent using ```kwargs.get(key,None)``` even it it's kinda redundant